### ISSUE TITLE
SR doesn't search for episodes which are 'wanted'

### ISSUE THREAD

**Issue by DDamian** [SickRage/sickrage-issues#21]
Ok, so this bug was in old SR and it still in newest one.
Old issue: https://github.com/SiCKRAGETV/sickrage-issues/issues/3603

I have multiple episoed as Wanted, and I can see they are already on TPB, but SR doesn't search for them, no word about it in debug logs.  When I manually click 'search' icon next to the episode, it snatch it immidiately.

Currently it is about NCIS S13E09:

```
2015-11-19 18:28:03 INFO     SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found
2015-11-19 18:28:03 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Transaction with 1 queries executed
2015-11-19 18:28:03 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Transaction with 1 queries executed
2015-11-19 18:28:03 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Unable to parse the filename [AnimeRG].Naruto.Shippuuden.-.438.[English.Subbed][480p].[KaMi] into a valid show
2015-11-19 18:28:03 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Attempting to add item to cache: [AnimeRG].Naruto.Shippuuden.-.438.[English.Subbed][480p].[KaMi]
2015-11-19 18:28:03 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Unable to parse the filename [HorribleSubs].Naruto.Shippuuden.-.438.[720p].mkv into a valid show
2015-11-19 18:28:02 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Attempting to add item to cache: [HorribleSubs].Naruto.Shippuuden.-.438.[720p].mkv
2015-11-19 18:28:02 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Unable to parse the filename [HorribleSubs].Naruto.Shippuuden.-.438.[480p].mkv into a valid show
2015-11-19 18:28:02 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Attempting to add item to cache: [HorribleSubs].Naruto.Shippuuden.-.438.[480p].mkv
2015-11-19 18:28:01 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Search URL: https://kickass.unblocked.pe/anime/?q=&field=time_add&rss=1&category=tv&sorder=desc
2015-11-19 18:28:01 INFO     POSTPROCESSER :: Processing completed
2015-11-19 18:28:01 DEBUG    POSTPROCESSER :: PostProcessing VideoInRar: []
2015-11-19 18:28:01 DEBUG    POSTPROCESSER :: PostProcessing RarContent: []
2015-11-19 18:28:01 DEBUG    POSTPROCESSER :: PostProcessing VideoFiles: []
2015-11-19 18:28:01 DEBUG    POSTPROCESSER :: PostProcessing Files: ['.DS_Store']
2015-11-19 18:28:01 DEBUG    POSTPROCESSER :: PostProcessing Dirs: ['@eaDir']
2015-11-19 18:28:01 INFO     POSTPROCESSER :: PostProcessing Path: /volume1/Other/Transmission/Complete
2015-11-19 18:28:01 DEBUG    POSTPROCESSER :: TV_DOWNLOAD_DIR: /volume1/Other/Transmission/Complete
2015-11-19 18:28:01 DEBUG    POSTPROCESSER :: Processing folder /volume1/Other/Transmission/Complete
2015-11-19 18:28:01 DEBUG    POSTPROCESSER :: Starting new thread: POSTPROCESSER
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Search URL: https://kickass.unblocked.pe/tv/?q=&field=time_add&rss=1&category=tv&sorder=desc
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Search URL: https://pirateproxy.pl/tv/latest?q=&orderby=7&category=200&type=search&page=0
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Search Mode: RSS
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Search Mode: RSS
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Narcos
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Fear the Walking Dead
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Mr. Robot
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from True Detective
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode NCIS S13E09
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from NCIS
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Sense8
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Zoo
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Safe House
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Game of Thrones
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Better Call Saul
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Shameless (US)
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from The Missing
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from The Affair
2015-11-19 18:28:00 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Halt and Catch Fire
2015-11-19 18:27:59 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from The Good Wife
2015-11-19 18:27:59 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Homeland
2015-11-19 18:27:59 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from The Last Ship
2015-11-19 18:27:59 INFO     SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes
```

**Comment by neoatomic** [comment]
@DDamian I'm trying to replicate your problem but without success so far.
When you click on the 'search' icon a backlog search is started. So that part works for you.
The Daily search basically only monitors the RSS feed
.
Could you try the following.? : 
  
* Rename the cache/rss folder, so SR will create a fresh one.
* Rename the cache.db so SR will create a fresh one.
* Rename your DB and create a fresh one with only that show.

![1](https://cloud.githubusercontent.com/assets/7928052/11300702/64402876-8f91-11e5-8e40-b85f587eb901.png)

**Comment by DDamian** [comment]
Points 1 & 2 were already done when I installed new SR on synology yesterday (so uninstalled old and installed new).

I will try with point 3 now.

**Comment by DDamian** [comment]
Removed db, started sr, added NCIS, only latest season set as Wanted. Been like this about 20 minutes, still all episodes are 'wanted'. I also manually started daily-search - no luck.

```
2015-11-20 14:57:20 INFO     SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found
2015-11-20 14:57:20 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Transaction with 9 queries executed
2015-11-20 14:57:20 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Transaction with 9 queries executed
2015-11-20 14:57:20 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Unable to parse the filename [Leopard-Raws].Lance.N'.Masques.-.08.RAW.(TBS.1280x720.x264.AAC).mp4 into a valid show
2015-11-20 14:57:19 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Attempting to add item to cache: [Leopard-Raws].Lance.N'.Masques.-.08.RAW.(TBS.1280x720.x264.AAC).mp4
2015-11-20 14:57:19 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Search URL: https://kickass.unblocked.pe/anime/?q=&field=time_add&rss=1&category=tv&sorder=desc
2015-11-20 14:57:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Search URL: https://kickass.unblocked.pe/tv/?q=&field=time_add&rss=1&category=tv&sorder=desc
2015-11-20 14:57:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Last update was too soon, using old cache: 2015-11-20 14:35:13. Updated less then 30 minutes ago
2015-11-20 14:57:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Search Mode: RSS
2015-11-20 14:57:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Agenci NCIS
2015-11-20 14:57:17 INFO     SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes
2015-11-20 14:57:14 INFO     DAILYSEARCHER :: No new released episodes found ...
2015-11-20 14:57:14 INFO     DAILYSEARCHER :: Searching for new released episodes ...
2015-11-20 14:57:14 INFO     Thread-26 :: Daily search forced
2015-11-20 14:54:28 INFO     SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Transaction with 9 queries executed
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Transaction with 9 queries executed
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Last update was too soon, using old cache: 2015-11-20 14:35:20. Updated less then 20 minutes ago
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Last update was too soon, using old cache: 2015-11-20 14:35:13. Updated less then 30 minutes ago
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode Agenci NCIS S13E09
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode Agenci NCIS S13E08
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode Agenci NCIS S13E07
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode Agenci NCIS S13E06
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode Agenci NCIS S13E05
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode Agenci NCIS S13E04
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode Agenci NCIS S13E03
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode Agenci NCIS S13E02
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: 72108: Loading episode details from DB for episode Agenci NCIS S13E01
2015-11-20 14:54:28 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Agenci NCIS
2015-11-20 14:54:28 INFO     SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes
2015-11-20 14:54:25 INFO     POSTPROCESSER :: Processing completed
2015-11-20 14:54:25 DEBUG    POSTPROCESSER :: PostProcessing VideoInRar: []
2015-11-20 14:54:25 DEBUG    POSTPROCESSER :: PostProcessing RarContent: []
2015-11-20 14:54:25 DEBUG    POSTPROCESSER :: PostProcessing VideoFiles: []
2015-11-20 14:54:25 DEBUG    POSTPROCESSER :: PostProcessing Files: ['.DS_Store']
2015-11-20 14:54:25 DEBUG    POSTPROCESSER :: PostProcessing Dirs: ['@eaDir']
2015-11-20 14:54:25 INFO     POSTPROCESSER :: PostProcessing Path: /volume1/Other/Transmission/Complete
2015-11-20 14:54:25 DEBUG    POSTPROCESSER :: TV_DOWNLOAD_DIR: /volume1/Other/Transmission/Complete
2015-11-20 14:54:25 INFO     DAILYSEARCHER :: No new released episodes found ...
2015-11-20 14:54:25 DEBUG    POSTPROCESSER :: Processing folder /volume1/Other/Transmission/Complete
2015-11-20 14:54:25 INFO     DAILYSEARCHER :: Searching for new released episodes ...
2015-11-20 14:49:49 DEBUG    Thread-62 :: git ls-remote --heads origin : returned successful
2015-11-20 14:49:49 DEBUG    Thread-62 :: Executing git ls-remote --heads origin with your shell in /usr/local/sickrage/var/SickRage
2015-11-20 14:49:49 DEBUG    Thread-62 :: git symbolic-ref -q HEAD : returned successful
2015-11-20 14:49:48 DEBUG    Thread-62 :: Executing git symbolic-ref -q HEAD with your shell in /usr/local/sickrage/var/SickRage
2015-11-20 14:49:48 DEBUG    Thread-62 :: git config remote.origin.url https://github.com/SickRage/SickRage.git : returned successful
2015-11-20 14:49:48 DEBUG    Thread-62 :: Executing git config remote.origin.url https://github.com/SickRage/SickRage.git with your shell in /usr/local/sickrage/var/SickRage
2015-11-20 14:44:26 DEBUG    CHECKVERSION :: check_for_new_news: Checking GitHub for latest news.
2015-11-20 14:44:26 DEBUG    CHECKVERSION :: cur_commit = ac5ccd7fec056ae974f9913f7685978dac2b1ec6, newest_commit = ac5ccd7fec056ae974f9913f7685978dac2b1ec6, num_commits_behind = 0, num_commits_ahead = 0
2015-11-20 14:44:26 DEBUG    CHECKVERSION :: git rev-list --left-right "@{upstream}"...HEAD : returned successful
2015-11-20 14:44:26 DEBUG    CHECKVERSION :: Executing git rev-list --left-right "@{upstream}"...HEAD with your shell in /usr/local/sickrage/var/SickRage
2015-11-20 14:44:26 DEBUG    CHECKVERSION :: git rev-parse --verify --quiet "@{upstream}" : returned successful
2015-11-20 14:44:26 DEBUG    CHECKVERSION :: Executing git rev-parse --verify --quiet "@{upstream}" with your shell in /usr/local/sickrage/var/SickRage
2015-11-20 14:44:26 DEBUG    CHECKVERSION :: git fetch origin : returned successful
2015-11-20 14:44:25 DEBUG    MAIN :: Transaction with 2271 queries executed
2015-11-20 14:44:25 DEBUG    CHECKVERSION :: Executing git fetch origin with your shell in /usr/local/sickrage/var/SickRage
2015-11-20 14:44:25 DEBUG    CHECKVERSION :: git config remote.origin.url https://github.com/SickRage/SickRage.git : returned successful
2015-11-20 14:44:25 INFO     POSTPROCESSER :: Processing completed
2015-11-20 14:44:25 DEBUG    POSTPROCESSER :: PostProcessing VideoInRar: []
2015-11-20 14:44:25 DEBUG    MAIN :: Internal name cache for Agenci NCIS set to: [ agenci ncis ]
2015-11-20 14:44:25 DEBUG    POSTPROCESSER :: PostProcessing RarContent: []
2015-11-20 14:44:25 DEBUG    CHECKVERSION :: Executing git config remote.origin.url https://github.com/SickRage/SickRage.git with your shell in /usr/local/sickrage/var/SickRage
2015-11-20 14:44:25 DEBUG    POSTPROCESSER :: PostProcessing VideoFiles: []
2015-11-20 14:44:25 DEBUG    CHECKVERSION :: git rev-parse HEAD : returned successful
2015-11-20 14:44:25 DEBUG    POSTPROCESSER :: PostProcessing Files: ['.DS_Store']
2015-11-20 14:44:25 DEBUG    MAIN :: Building internal name cache for Agenci NCIS
2015-11-20 14:44:25 DEBUG    POSTPROCESSER :: PostProcessing Dirs: ['@eaDir']
2015-11-20 14:44:25 DEBUG    MAIN :: No scene exceptions update needed
2015-11-20 14:44:25 INFO     POSTPROCESSER :: PostProcessing Path: /volume1/Other/Transmission/Complete
2015-11-20 14:44:25 DEBUG    CHECKVERSION :: Executing git rev-parse HEAD with your shell in /usr/local/sickrage/var/SickRage
2015-11-20 14:44:25 DEBUG    CHECKVERSION :: git symbolic-ref -q HEAD : returned successful
2015-11-20 14:44:24 INFO     MAIN :: Building internal name cache for all shows
2015-11-20 14:44:24 INFO     FINDSUBTITLES :: No subtitles to download
2015-11-20 14:44:24 DEBUG    MAIN :: No scene exceptions update needed
2015-11-20 14:44:24 DEBUG    POSTPROCESSER :: TV_DOWNLOAD_DIR: /volume1/Other/Transmission/Complete
2015-11-20 14:44:24 INFO     FINDSUBTITLES :: Checking for subtitles
2015-11-20 14:44:24 DEBUG    POSTPROCESSER :: Processing folder /volume1/Other/Transmission/Complete
2015-11-20 14:44:24 DEBUG    FINDSUBTITLES :: Starting new thread: FINDSUBTITLES
2015-11-20 14:44:24 DEBUG    POSTPROCESSER :: Starting new thread: POSTPROCESSER
2015-11-20 14:44:24 DEBUG    CHECKVERSION :: Executing git symbolic-ref -q HEAD with your shell in /usr/local/sickrage/var/SickRage
2015-11-20 14:44:24 DEBUG    CHECKVERSION :: Starting new thread: CHECKVERSION
2015-11-20 14:44:24 INFO     TORNADO :: Starting SickRage on http://0.0.0.0:8899/
```

![2015-11-20_1500](https://cloud.githubusercontent.com/assets/939650/11301443/79c3820a-8f97-11e5-9884-61b8bb6f42a9.png)

**Comment by DDamian** [comment]
```
2015-11-20 15:07:17 INFO     SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found
2015-11-20 15:07:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Transaction with 9 queries executed
2015-11-20 15:07:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Transaction with 9 queries executed
2015-11-20 15:07:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Last update was too soon, using old cache: 2015-11-20 14:57:19. Updated less then 20 minutes ago
2015-11-20 15:07:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Search URL: https://pirateproxy.pl/tv/latest?q=&orderby=7&category=200&type=search&page=0
2015-11-20 15:07:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Search Mode: RSS
2015-11-20 15:07:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Agenci NCIS
2015-11-20 15:07:17 INFO     SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes
```

https://pirateproxy.pl/tv/latest?q=&orderby=7&category=200&type=search&page=0   doesn't seem to be working

**Comment by DDamian** [comment]
Another 2 searches, all episodes still 'wanted'
```
2015-11-20 15:27:20 INFO     SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found
2015-11-20 15:27:20 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Transaction with 9 queries executed
2015-11-20 15:27:20 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Transaction with 9 queries executed
2015-11-20 15:27:20 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Unable to parse the filename [Leopard-Raws].Lance.N'.Masques.-.08.RAW.(TBS.1280x720.x264.AAC).mp4 into a valid show
2015-11-20 15:27:19 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Attempting to add item to cache: [Leopard-Raws].Lance.N'.Masques.-.08.RAW.(TBS.1280x720.x264.AAC).mp4
2015-11-20 15:27:18 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Search URL: https://kickass.unblocked.pe/anime/?q=&field=time_add&rss=1&category=tv&sorder=desc
2015-11-20 15:27:18 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Search URL: https://kickass.unblocked.pe/tv/?q=&field=time_add&rss=1&category=tv&sorder=desc
2015-11-20 15:27:18 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Last update was too soon, using old cache: 2015-11-20 15:07:17. Updated less then 30 minutes ago
2015-11-20 15:27:18 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Search Mode: RSS
2015-11-20 15:27:18 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Agenci NCIS
2015-11-20 15:27:18 INFO     SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes
2015-11-20 15:27:15 INFO     DAILYSEARCHER :: No new released episodes found ...
2015-11-20 15:27:15 INFO     DAILYSEARCHER :: Searching for new released episodes ...
2015-11-20 15:24:26 INFO     POSTPROCESSER :: Processing completed
2015-11-20 15:24:26 DEBUG    POSTPROCESSER :: PostProcessing VideoInRar: []
2015-11-20 15:24:26 DEBUG    POSTPROCESSER :: PostProcessing RarContent: []
2015-11-20 15:24:26 DEBUG    POSTPROCESSER :: PostProcessing VideoFiles: []
2015-11-20 15:24:26 DEBUG    POSTPROCESSER :: PostProcessing Files: ['.DS_Store']
2015-11-20 15:24:26 DEBUG    POSTPROCESSER :: PostProcessing Dirs: ['@eaDir']
2015-11-20 15:24:26 INFO     POSTPROCESSER :: PostProcessing Path: /volume1/Other/Transmission/Complete
2015-11-20 15:24:26 DEBUG    POSTPROCESSER :: TV_DOWNLOAD_DIR: /volume1/Other/Transmission/Complete
2015-11-20 15:24:26 DEBUG    POSTPROCESSER :: Processing folder /volume1/Other/Transmission/Complete
2015-11-20 15:24:26 DEBUG    POSTPROCESSER :: Starting new thread: POSTPROCESSER
2015-11-20 15:17:18 INFO     SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found
2015-11-20 15:17:18 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Transaction with 9 queries executed
2015-11-20 15:17:18 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Transaction with 9 queries executed
2015-11-20 15:17:18 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Last update was too soon, using old cache: 2015-11-20 14:57:19. Updated less then 20 minutes ago
2015-11-20 15:17:18 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [ThePirateBay] :: Last update was too soon, using old cache: 2015-11-20 15:07:17. Updated less then 30 minutes ago
2015-11-20 15:17:17 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from Agenci NCIS
2015-11-20 15:17:17 INFO     SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes
2015-11-20 15:17:15 INFO     DAILYSEARCHER :: No new released episodes found ...
2015-11-20 15:17:15 INFO     DAILYSEARCHER :: Searching for new released episodes ...
```

**Comment by Fetchered** [comment]
@DDamian Are you using a VPN service? I had the same problem over the last 24 hours. I was connected via VPN and my provider went down briefly. Once restored, I couldn't seem to successfully search for or snatch anything. I tried all three solutions above but they didn't work. After I disconnected, reconnected again to my VPN (with a reboot of my cable modem), and a reboot of SickRage, it seems to be working for me again. 

If you're not using a VPN, then the reboot and manual search might work. I know it sounds cliche (have you tried turning it off and on again), but this did work for me.

**Comment by DDamian** [comment]
No, I don't use VPN.

**Comment by miigotu** [comment]
Solved in IRC, he had backlog set to 10080 minutes -.-

**Comment by NathanSweet** [comment]
I have these same symptoms. What does it mean to set the backlog to XXX minutes?
